frontend/src/services/transactions/request-manager.ts

```diff
@@ -14,13 +14,18 @@ export async function getRequestFee(
   const connectedContract = ethereumProvider.connectContract(requestManagerContract);
   return await connectedContract.totalFee();
 }
+interface TransactionReceiptInterface extends TransactionReceipt {
+  events: Array<any>;
```

Comment:
Is this really working? According to the ethers documentation, there should be a `logs` property on the `TransactionReceipt`.

Reply:
`logs` doesn't contain events information, `events`  property does in its `args` and unfortunately it isn't available in  `TransactionReceipt` interface!